### DISCUSSION TITLE
ncat fails

### DISCUSSION THREAD

**Discussion by chinese-maintainer** [bol-van/zapret#865] [Q&A]
На OpenWrt blockcheck выдает все-время непонятную ошибку
```
104.16.43.196 connects
ncat -z -w 2 104.16.41.196 443
libnsock ssl_init_helper(): OpenSSL legacy provider failed to load.
```

**Comment by bol-van** [comment]
Это наверняка проблема ncat или его сборки.
Если выполнить команду вручную, тоже самое ?

**Comment by chinese-maintainer** [comment]
Да, то же самое

**Comment by bol-van** [comment]
В openwrt есть ncat,ncat-full,ncat-ssl.
Если какой-то вариант со сломанным SSL, надо использовать вариант просто ncat. Он без SSL.
К zapret это отношения не имеет, а имеет к сборщиками openwrt.
Судя по сообщению ошибка не критическая и связана с отсутствием какого-то внешнего модуля openssl, который выкинули для экономии места

**Comment by chinese-maintainer** [comment]
Я поставил просто ncat и теперь все нормально
